src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/IntegrationTestBase.java

```diff
@@ -61,6 +66,22 @@ public void setup() {
         connectionSettings = PropertyUtil.readFormVmArgs() != null ?
                 PropertyUtil.readFormVmArgs() : PropertyUtil.readFromPropFile();
 
+        Annotation[] ants = this.getClass().getDeclaredAnnotations();
+
+
+        for (Annotation annotation : ants) {
+            if (annotation.toString().contains("WorkSpace(clean=")) {
```

Comment:
Annotation is the base, it can be cast to your specific type, to string comparing this is not ok
Use annotation instance of YourAnnotationClass
After which cast the annotation to your specific annotations class, and retreive the clean boolan